pom.xml

```diff
@@ -243,10 +243,9 @@
             <version>1.0.24</version>
         </dependency>
         <dependency>
-            <groupId>org.kohsuke</groupId>
-            <artifactId>trilead-putty-extension</artifactId>
-            <version>1.2</version>
-            <scope>runtime</scope>
+            <groupId>com.google.code.findbugs</groupId>
```

Comment:
these come in via the parent pom, in the form of `[com.github.spotbugs](com.github.spotbugs:spotbugs-annotations)` so I don't think we need it here